app/controllers/groups_controller.rb

```diff
@@ -127,7 +127,7 @@ def assign_scans
 
   def get_names
     names = Student
-              .select(:id, "CONCAT(first_name,' ',last_name) AS label, CONCAT(first_name,' ',last_name) AS value")
+              .select(:id, :id_number, "CONCAT(first_name,' ',last_name) AS label, CONCAT(first_name,' ',last_name) AS value")
               .where("(lower(first_name) like ? OR lower(last_name) like ?) AND users.id NOT IN (?)",
```

Comment:
Wait okay I think we miscommunicated. I still want you to *search* the `id_number` column; it's just that this column is now a string, so you can just use a plain `LIKE`.